src/PerformanceTests/Common/EnvironmentStats.cs

```diff
@@ -31,5 +32,11 @@ public static void Write()
         log.InfoFormat("EnableDnsRoundRobin: {0}", ServicePointManager.EnableDnsRoundRobin);
         log.InfoFormat("Expect100Continue: {0}", ServicePointManager.Expect100Continue);
         log.InfoFormat("UseNagleAlgorithm: {0}", ServicePointManager.UseNagleAlgorithm);
+
+        log = LogManager.GetLogger("ENVVAR");
```

Comment:
the GetLogger(string) api is not available when targeting netstandard. Can you switch that, e.g. see https://github.com/Particular/EndToEnd/pull/281